geomesa-accumulo/geomesa-accumulo-tools/src/main/scala/org/locationtech/geomesa/accumulo/tools/AccumuloRunner.scala

```diff
@@ -90,14 +95,16 @@ object AccumuloRunner extends Runner {
     }.foreach(_.password = Prompt.readPassword())
 
     params.collect { case p: InstanceNameParams => p }.foreach { p =>
-      if (p.zookeepers == null) {
-        p.zookeepers = zookeepers
-      }
-      if (p.instance == null) {
-        p.instance = try {
+      if (! p.mock) {
+        if (p.zookeepers == null) {
+          p.zookeepers = zookeepers
+        }
+
```

Comment:
I'm not really seeing the same behavior here. Here are my test results.

Without ACCUMULO_HOME set or classpath additions:
-z and -i worked
-z only failed
-i only failed

With ACCUMULO_HOME set:
-z and -i worked
-z only worked 
-i only worked

Without ACCUMULO_HOME set but with classpath additions (no accumulo-site.xml):
-z and -i worked
-z only worked
-i only failed

Without ACCUMULO_HOME set but with classpath additions and geomesa.tools.accumulo.site.xml set:
-z and -i worked
-z only worked
-i only worked

These results seem to make sense to me.